lua/dashboard/init.lua

```diff
@@ -220,15 +209,34 @@ function db:load_theme(opts)
     end,
   })
 
+  api.nvim_create_autocmd('BufEnter', {
+    buffer = self.bufnr,
```

Comment:
can be once ?

Reply:
Oh yeah, merged with the other BufEnter

Reply:
Not sure why CI is failing, but let me know what I can do to fix it.

Reply:
nvm it just doc.